docs/excel/excel-add-ins-shapes.md

```diff
@@ -0,0 +1,237 @@
+    wave.top = 400;
+    wave.height = 50;
+    wave.width = 150;
+    wave.name = "Wave";
+    wave.fill.setSolidColor("lightblue");
+    wave.textFrame.textRange.text = "Shape text";
+    wave.textFrame.textRange.font.color = "purple";
+    wave.textFrame.horizontalAlignment = Excel.ShapeTextHorizontalAlignment.center;
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+The `addTextBox` method of `ShapeCollection` creates a `GeometricShape` of type `Rectangle` with a white background and black text. This is the same as what is created by Excel's **Text Box** button on the **Insert** tab. `addTextBox` takes a string argument to set the text of the `TextRange`.
+
+The following code sample shows the creation of a text box with the text "Hello!".
+
+```js
+// This sample creates a text box with the text "Hello!" and sizes it appropriately.
+Excel.run(function (context) {
+    var shapes = context.workbook.worksheets.getItem("MyWorksheet").shapes;
+    var textbox = shapes.addTextBox("Hello!");
+    textbox.left = 100;
+    textbox.top = 100;
+    textbox.height = 20;
+    textbox.width = 45;
+    textbox.name = "Textbox";
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+## Shape groups
+
+Shapes can be grouped together. This allows a user to treat them as a single entity. A [ShapeGroup](/javascript/api/excel/excel.shapegroup) is a type of `Shape`, so your add-in treats the group as a single shape.
+
+The following code sample shows three shapes being grouped together. The subsequent code sample shows that shape group being moved to the right 50 pixels.
+
+```js
+// This sample takes three previously-created shapes ("Square", "Pentagon", and "Octagon")
+// and groups them into a single ShapeGroup.
+Excel.run(function (context) {
+    var shapes = context.workbook.worksheets.getItem("MyWorksheet").shapes;
+    var square = shapes.getItem("Square");
+    var pentagon = shapes.getItem("Pentagon");
+    var octagon = shapes.getItem("Octagon");
+
+    var shapeGroup = shapes.addGroup([square, pentagon, octagon]);
+    shapeGroup.name = "Group";
+    console.log("Shapes grouped");
+
+    return context.sync();
+}).catch(errorHandlerFunction);
+
+// This sample moves the previously created shape group to the right by 50 pixels.
+Excel.run(function (context) {
+    var shapes = context.workbook.worksheets.getItem("MyWorksheet").shapes;
+    var shapeGroup = sheet.shapes.getItem("Group");
+    shapeGroup.incrementLeft(50);
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+Individual shapes within the group are referenced through the `ShapeGroup.shapes` property, which is of type [GroupShapeCollection](/javascript/api/excel/excel.GroupShapeCollection). They are no longer accessible through the worksheet's shape collection after being grouped. As an example, if your worksheet had three shapes and they were all grouped together, the worksheet's `shapes.getCount` method would return a count of 1.
```

Comment:
This paragraph seems like an important gotcha. I suggest putting it in an Important! note.